### ISSUE TITLE
How can I reproduce the results in the paper?

### ISSUE THREAD

**Issue by jinwchoi** [hbilen/dynamic-image-nets#4]
How can I reproduce the results in the paper?
I could train the dynamic image net.
However, I could not find prediction or evaluation script in the code.
Could you please tell me how to do the evaluation? Or can you upload the evaluation code?

**Comment by hbilen** [comment]
I have pushed an evaluation script. Please have a look at README file

**Comment by jinwchoi** [comment]
Hi,

I tried your evaluation code, but it looks like a training code.
The result I got is shown below.
----------------------------------------------------------------------------------
>> info = cnn_dicnn_evaluate()    
Warning: Name is nonexistent or not a directory: Datasets 
> In path at 109
  In addpath at 86
  In cnn_dicnn_evaluate at 8 

ans = 

  CUDADevice with properties:

                      Name: 'Tesla K80'
                     Index: 8
         ComputeCapability: '3.7'
            SupportsDouble: 1
             DriverVersion: 7.5000
            ToolkitVersion: 5.5000
        MaxThreadsPerBlock: 1024
          MaxShmemPerBlock: 49152
        MaxThreadBlockSize: [1024 1024 64]
               MaxGridSize: [2.1475e+09 65535 65535]
                 SIMDWidth: 32
               TotalMemory: 1.2885e+10
                FreeMemory: 1.2637e+10
       MultiprocessorCount: 13
              ClockRateKHz: 823500
               ComputeMode: 'Default'
      GPUOverlapsTransfers: 1
    KernelExecutionTimeout: 0
          CanMapHostMemory: 1
           DeviceSupported: 1
            DeviceSelected: 1

cnn_train_dag: resuming by loading epoch 67

info = 

    train: [1x67 struct]
      val: [1x67 struct]

----------------------------------------------------------------------------------

I think cnn_dicnn_evaluate.m file calls cnn_train_dag() function not a testing function.
Could you please share the code to get an accuracy for the testset so that I can reproduce the numbers in your paper? It would be really helpful.

**Comment by hbilen** [comment]
Hi Jinwoo,

Training and evaluation code are same indeed. info.val.mcerr should 
already be giving you the result.

If you want to try the model that I share, please copy the model into an 
empty folder (e.g. /tmp/1) and choose opts.expDir = '/tmp/1/' and 
opts.modelPath='/tmp/1/the-model-file' and try again the evaluation code.

Please note that I already share all the code to train and evaluate and 
don't forget that this is only a research code, you may need to dirty 
your hands :)

Best,

Hakan
